NodeManagerLibrary.ino

```diff
@@ -633,8 +633,15 @@ void Sensor::loop(MyMessage* message) {
       _track_last_value && child->isNewValue() || 
       _track_last_value && child->force_update_timer->isRunning() && child->force_update_timer->isOver()
       ) 
-#endif
         child->sendValue();
+#else      
+      //recive OR send message
+      // we've been called from NOT from receive(), pass the message along
+      if (message == nullptr)
+        child->sendValue();
```

Comment:
Not sure about this change. The node should report back either if executing loop() or if called from receive(). But you probably need to add a _report_timer->unset(); in onSetup() so to report immediately like a relay

Reply:
Hi, 
I pushed this change after PK, and I thought that it would not enter it.

I find it somewhat strange that the node, after receiving the message, sends something back. This is not the simple ACK, this is the "same" data.

If we want to set "every" for a 300-led stripe, this generates "extra" 300 messages, in which the same message comes. If we want to sign messages - two messages of additional messages. Total 3 additional messages per point. 

On STM32 this did not work as it should - error message in serial log after 2 seconds delay.

I do not insist on this change, but I find this behavior strange

Reply:
I understand this could not be ideal, but just to keep it aligned and coherent with the other sensors. Then if a user is afraid of too many messages (which is reasonable), can customize the code but as a default I'd keep the original behavior. This is mainly because new users, when they send a message and see nothing back can wonder if the message was formatted correctly. Having it echoed back ensure the routine is executed. Also, some controllers would expect this kind of communication, maybe not for this specific sensor but for others (e.g. a relay). 
If you can remove "if (message == nullptr)" I'll merge this PR into the branch. Thanks again!

Reply:
Hi,
"if (message == nullptr)" now removed